Contributing.md

```diff
@@ -1,37 +1,79 @@
-
-## TECHNOLOGIES
-* Programming Language - Java
-* Issue Tracking - GitHub
-* Version Control - Git
-* Build Process - Apache Ant 
-
-## TOOLS
-### Git
-* [Git](https://git-scm.com)  
-* [Git Clients](https://git-scm.com/downloads)
-
-### GitHub
-* [GitHub](https://github.com/)
-* [GitHub Tutorial](https://guides.github.com/activities/hello-world/) - "Hello World" starter guide for those new to GitHub
-
-### Apache Ant 
-* [Apache Ant](https://ant.apache.org/bindownload.cgi)
-
-#### Using Ant at the command line 
-```
-   $ cd <directory containing build.xml>
-   $ ant all 
-   $ cd build
-```
+# Contributing to the Redistricting project
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## PROJECT DESCRIPTION
+Please refer to [README.md](README.md)
+
+## PROJECT WORKFLOW
+Please refer to the [Project Kanban Board](https://github.com/MetroCS/redistricting/projects/1)
+
+## PROJECT DISCUSSIONS
+This project uses [Discussions](https://github.com/MetroCS/redistricting/discussions) for project-related announcements, questions and answers, ideas, and engaging in all discussions not related to a specific project [Issue](https://github.com/MetroCS/redistricting/issues) or [Pull Request](https://github.com/MetroCS/redistricting/pulls).
+
+## TECHNOLOGIES
+* Programming Language - Java
+* Issue Tracking - GitHub
+* Version Control - Git
+* Build Process - Apache Ant 
+
+## TOOLS
+
+### Git
+* [Git](https://git-scm.com)  
+* [Git Clients](https://git-scm.com/downloads)
+
+### GitHub
+* [GitHub](https://github.com/)
+* [GitHub Tutorial](https://guides.github.com/activities/hello-world/) - "Hello World" starter guide for those new to GitHub
+
+### Apache Ant: Version 1.10.7
+* Note: Version 1.10.7 does not allow the user to execute
+```
+   $ ant test
+
+```
+*  However, Version 1.10.12 does allow the test to run.
+* [Apache Ant](https://ant.apache.org/bindownload.cgi)
+
+#### Using Ant at the command line 
+```
+   $ cd <directory containing build.xml>
+   $ ant all 
+   $ cd build
+
+```
+### Java: Version 11.0.14.1
+* [Java](https://www.java.com/en/)
+
+### Junit: Version 1.5.1
+* [Junit](https://junit.org/)
+
+### Jacoco: Version 0.8.2
+* [Jacoco](https://www.eclemma.org/jacoco/)
+
+### Checkstyle: Version 7.5
+* [Checkstyle](https://checkstyle.sourceforge.io/)
+
+### PMD, CPD: 5.5.2
+* [PMD](https://pmd.github.io/latest/pmd_userdocs_cpd.html)
+
+### ProGuard: Version 6.2.2
+* [Proguard](https://www.guardsquare.com/proguard)
+
+### Spotbugs: 4.5.2
+* [Spotbugs](https://spotbugs.github.io/)
+
+### Java2HTML: Version 5.0
+* [Java2HTML](http://www.java2html.com/)
+
+### The following are used by tools in the build:
+
+#### Gson: Version 2.8.5
+* [GSon](https://github.com/google/gson)
```

Comment:
If this knowledge is important, it seems like it is also important to know which tools use it.
Likewise for each of the tools only used by other tools and not directly invoked by the build itself.